autoapi/templates/python/function.rst

```diff
@@ -1,6 +1,8 @@
 {% if obj.display %}
 .. function:: {{ obj.short_name }}({{ obj.args }}){% if obj.return_annotation is not none %} -> {{ obj.return_annotation }}{% endif %}
 
+   {# Force Jinja to honor a newline here #}
+   {{''}} 
```

Comment:
I am being picky but with what's here I'm getting some trailing whitespace in the generated rst. Ideally the rst output is as human friendly as possible so this might bother some users.
I've used a double empty line in the other template files to get the spacing to look like we want. Some quick testing looks like that should work here too?

Reply:
Feel free to use another approach, I just couldn't get them working locally.